systems/robotInterfaces/QPControllerPlan.m

```diff
@@ -1,4 +1,4 @@
-classdef QPControllerPlan
+classdef QPControllerPlan < handle
```

Comment:
that's a pretty big change.  are you sure you've captured all the implications of this?